neanderthal/src/main/kotlin/au/com/outware/neanderthal/presentation/adapter/PropertyAdapter.kt

```diff
@@ -40,19 +40,13 @@ class PropertyAdapter : RecyclerView.Adapter<RecyclerView.ViewHolder>(),
     override fun setItem(variant: Variant) {
         this.variant = variant
 
-        val fields = variant.configuration!!.javaClass.declaredFields.filter {
-            field -> !Modifier.isPrivate(field.modifiers) && !Modifier.isTransient(field.modifiers)
-        }.sortedBy { selector -> selector.name }
-        properties.addAll(fields)
-        for(property in properties) {
-            property.isAccessible = true
-        }
+        properties.addAll(variant.configuration!!.javaClass.serializableFields.sortedBy { selector -> selector.name })
```

Comment:
Never mind, I saw the serializableFields class at the bottom of this PR. Nice one